src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/CreateVMFooter.tsx

```diff
@@ -13,12 +23,63 @@ import {
 
 import './CreateVMFooter.scss';
 
-const CreateVMFooter: FC = () => {
+type CreateVMFooterProps = {
+  vm: V1VirtualMachine;
+  onCancel: () => void;
+  bootableVolumeSelected: V1beta1DataSource;
```

Comment:
Sorry. I missed this one yesterday. We should make this `selectedBootableVolume` as well.

Reply:
great catch :) 
fixed